envs/example/ci-ceph_swift/group_vars/all.yml

```diff
@@ -20,3 +30,41 @@ cinder:
     # Ceph
     - name: rbd_volumes
       volume_driver: cinder.volume.drivers.rbd.RBDDriver
+
+swift:
+  enabled: True
+  allow_versions: True
+  disks:
+    - disk: vdb
```

Comment:
That style with a simple list worked before, but it will cause an error with latest code of swift roles in ursula. So the way I am using is just to fit the latest ursula code. : ) 
Btw, you can also see that format here which was updated 13 days ago: https://github.com/blueboxgroup/ursula/blob/master/envs/example/swift/group_vars/all.yml